exercises/two-fer/.meta/generator/two_fer_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class TwoFerCase < Generator::ExerciseCase
+
+  def workload
+    assert_equal { "TwoFer.two_fer" }
```

Comment:
This generator does not generate the test cases.

See: https://github.com/exercism/ruby#generated-test-suites